src/Application.vala

```diff
@@ -109,6 +109,10 @@ namespace Tooth {
 				error (msg);
 			}
 
+			var style_manager = Adw.StyleManager.get_default ();
```

Comment:
Here the value from the settings is obtained as a ToothColorScheme, and its adwaita equivalent is given to the style manager.